src/components/navbar/navbar.tsx

```diff
@@ -41,19 +40,21 @@ export const Navbar: FC<Props> = ({ open, setOpen }) => {
             {loggedUser.data?.username ?? text.general.username}
           </Typography>
         </Stack>
-        <Button
-          variant="text"
-          sx={styles.mainLogo}
-          onClick={() => handleNavigation(pages.home)}
-        >
-          <BotBustersIcon />
-        </Button>
-        <Stack direction={"row"} rowGap={2} sx={styles.navbarEnd}>
+        {!isHomePage && (
+          <Button
+            variant="text"
+            sx={styles.mainLogo}
+            onClick={() => handleNavigation(pages.home)}
+          >
+            <BotBustersIcon />
+          </Button>
+        )}
+        <Stack direction={"row"} rowGap={4} sx={styles.navbarEnd}>
           <AudioSettings />
-          <MenuButton sx={styles.button} onClick={() => setOpen(true)} />
+          <MenuButton sx={styles.button} onClick={() => setMenuIsOpen(true)} />
         </Stack>
       </Stack>
-      <MainMenu open={open} setOpen={setOpen} />
+      <MainMenu open={menuIsOpen} setOpen={setMenuIsOpen} />
```

Comment:
Conventionally you would always call this booleans "isOpenMenu", with the format "is<yourCondition>: